WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -202,7 +207,20 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 
 			this.WhenAnyValue(x => x.FeeTarget)
 				.ObserveOn(RxApp.MainThreadScheduler)
-				.Subscribe(_ => SetFeesAndTexts());
+				.Subscribe(_ =>
+				{
+					IsSliderFeeUsed = true;
+					SetFeesAndTexts();
+				}
+				);
```

Comment:
Usually the closing parenthesis is written in the same line as the closing curly brace.